2-copy-of-code/lesson-16/tests/checkout/orderSummaryTest.js

```diff
@@ -43,6 +43,12 @@ describe('test suite: renderOrderSummary', () => {
     expect(
       document.querySelector(`.js-product-quantity-${productId2}`).innerText
     ).toContain('Quantity: 1');
+    expect(
+      document.querySelector(`.js-product-name-${productId1}`).innerText
+    ).toEqual('Black and Gray Athletic Cotton Socks - 6 Pairs');

```

Comment:
I saved the names of the products in variables and used them later : `   const productName1 = 'Black and Gray Athletic Cotton Socks - 6 Pairs';
    const productName2 = 'Intermediate Size Basketball';

    expect(document.querySelector(`.js-product-name-${productId1}`).innerText).toContain(productName1);
`